lib/testsuite/cucumber.js

```diff
@@ -0,0 +1,91 @@
+const Cli =  require('@cucumber/cucumber/lib/cli').default;
```

Comment:
Since cucumber will be an optional dependency, this needs to be wrapped in a try/catch. Also, don't they expose a standard programatic api?

Reply:
No, I don't think they expose programmatic API